src/pysdmx/model/vtl.py

```diff
@@ -88,11 +109,30 @@ class CustomType(Item, frozen=True, omit_defaults=True):
     vtl_scalar_type: str = ""
 
 
+class _VTLVersionEnum(Enum):
+    """Enumeration of VTL versions."""
+
+    VTL_1_0 = "1.0"
+    VTL_2_0 = "2.0"
+    VTL_2_1 = "2.1"
+
+
 class VtlScheme(ItemScheme, frozen=True, omit_defaults=True):
     """A VTL item scheme with the additional propery 'vtl_version'."""
 
     vtl_version: Optional[str] = None
 
+    def __post_init__(self) -> None:
```

Comment:
There are other VTL artefacts which expect a vtl_version, unless I'm mistaken. Should a similar check be added there too?

Also, should vtl_version be an enum then, instead of a str (just a question)?

Reply:
I guess vtl_version could be either a enum or a string, but I do prefer to leave it as string until we find proper use cases. The vtlVersion property is just on the 5 classes it is already added, with inheritance to VtlScheme.